server/src/main/java/org/elasticsearch/action/admin/indices/analyze/TransportAnalyzeAction.java

```diff
@@ -65,6 +66,8 @@
  */
 public class TransportAnalyzeAction extends TransportSingleShardAction<AnalyzeAction.Request, AnalyzeAction.Response> {
 
+    public static final NodeFeature WRONG_CUSTOM_ANALYZER_RETURN_400 = new NodeFeature("wrong_custom_analyzer_return_400");
```

Comment:
Could you simply make this a capability on the REST API side? That is much simpler and then you don't need all these feature things.